src/sql/workbench/parts/dataExplorer/browser/dataExplorerExtensionPoint.ts

```diff
@@ -133,15 +133,15 @@ class DataExplorerContainerExtensionHandler implements IWorkbenchContribution {
 
 		for (let descriptor of viewDescriptors) {
 			if (typeof descriptor.id !== 'string') {
-				collector.error(localize('requirestring', "property `{0}` is mandatory and must be of type `string`", 'id'));
+				collector.error(localize('requirestring', "property `{0}` is mandatory and must be of type `string`", "id"));
 				return false;
 			}
 			if (typeof descriptor.name !== 'string') {
-				collector.error(localize('requirestring', "property `{0}` is mandatory and must be of type `string`", 'name'));
+				collector.error(localize('requirestring', "property `{0}` is mandatory and must be of type `string`", "name"));
 				return false;
 			}
 			if (descriptor.when && typeof descriptor.when !== 'string') {
-				collector.error(localize('optstring', "property `{0}` can be omitted or must be of type `string`", 'when'));
+				collector.error(localize('optstring', "property `{0}` can be omitted or must be of type `string`", "when"));
```

Comment:
So this one indicates an error in the logic. It's localize(id, localizableString, ...args). In this case it's marking the args as the things that need "" which isn't correct.

Reply:
Hm, shoot. Yeah that's a tricky one. I'll poke around but I don't see a simple way to solve that without getting into some pretty crazy regex land which I was trying to avoid.

For now would you see a problem with just leaving this as changed? If it starts being a huge annoyance we can look into making the regex more robust but right now I'm inclined to just do this amount and then come back to it if needed.